packages/react-core/src/components/NotificationBadge/NotificationBadge.tsx

```diff
@@ -2,27 +2,49 @@ import * as React from 'react';
 import { Button, ButtonVariant, ButtonProps } from '../Button';
 import { css } from '@patternfly/react-styles';
 import styles from '@patternfly/react-styles/css/components/NotificationBadge/notification-badge';
+import AttentionBellIcon from '@patternfly/react-icons/dist/js/icons/attention-bell-icon';
+import BellIcon from '@patternfly/react-icons/dist/js/icons/bell-icon';
 
-export interface NotificationBadgeProps extends ButtonProps {
+export enum NotificationBadgeVariant {
+  read = 'read',
+  unread = 'unread',
+  attention = 'attention'
+}
+
+export interface NotificationBadgeProps extends Omit<ButtonProps, 'variant'> {
   /**  Adds styling to the notification badge to indicate it has been read */
   isRead?: boolean;
```

Comment:
deprecate this in favor of variant?

Reply:
I'm just trying to avoid breaking changes. is there a best way to deprecate it? Should I add a warning to anyone using it?

Reply:
maybe in the docs comment add 
`/** @deprecated Use the variant prop instead - Adds styling to the notification badge to indicate it has been read */`
and in code
```
// when you destructure isRead don't give it the default value of false
if (process.env.NODE_ENV !== 'production' && isRead !== undefined) {
  console.warn(
        'The isRead prop has been deprecated. Use the variant prop instead.'
      );
}
```